README.md

```diff
@@ -1 +1,87 @@
-# fuelstop-examples
+# FuelStop
+
+## Getting Started
+
+#### Projects
+
+After sign up, you can find your projects by clicking on the menu icon on the top-left corner of the page. One project is for the Ethereum Mainnet and the other is for Ropsten (this is the only testnet that's currently supported). Clicking on either of the projects will bring you to its details page.
+
+![menu](./diagrams/projects.svg)
+
+#### Project Details
+
+The `Project Details` page includes the project's balance, registered addresses, transaction history, account history, and the unique JSON-RPC URL containing the Project ID which is used to authenticate your API calls.
+
+###### Balance
+
+The `Balance` section is where you can add your credit card to your FuelStop account. After the card is added, its info is displayed. You can also remove a card once it has been added. The `Mainnet` project starts without credit-card so, you have to add one before your transactions can be paid for. The `Ropsten` project, however, has a test-card already registered that cannot be removed. Feel free to hit `Top up` as much as you want with this project, you won't be charged.
+
+![Balance without credit card](./diagrams/balance-without-credit-card.svg)
+
+![FuelStop Documentation](./diagrams/balance.svg)
+
+###### Details
+
+Here, you can find the name of the network and your RPC URL. FuelStop's Ethereum JSON-RPC APIs require a valid project ID to be part of your URL to authenticate the calls. Do not share the URL.
+
+![Details](./diagrams/details.svg)
+
+###### Addresses
+
+Before sending your signed transactions (via the JSON-RPC `eth_sendRawTransaction` method or with a client-library), you need to register your addresses. If you send FuelStop a signed transaction that is not included in your address-list, FuelStop will behave no differently than a regular Ethereum node and will broadcast it to the network. If your address has some ETH but not enough to cover the transaction's fees, FuelStop will fund your address so that the transaction can go through.
+
+![Addresses](./diagrams/addresses.svg)
+
+###### Transactions
```

Comment:
```suggestion
#### Balance

The `Balance` section is where you can add your credit card to your FuelStop account. After the card is added, its info is displayed. You can also remove a card once it has been added. The `Mainnet` project starts without credit-card so, you have to add one before your transactions can be paid for. The `Ropsten` project, however, has a test-card already registered that cannot be removed. Feel free to hit `Top up` as much as you want with this project, you won't be charged.

![Balance without credit card](./diagrams/balance-without-credit-card.svg)

![FuelStop Documentation](./diagrams/balance.svg)

#### Details

Here, you can find the name of the network and your RPC URL. FuelStop's Ethereum JSON-RPC APIs require a valid project ID to be part of your URL to authenticate the calls. Do not share the URL.

![Details](./diagrams/details.svg)

#### Addresses

Before sending your signed transactions (via the JSON-RPC `eth_sendRawTransaction` method or with a client-library), you need to register your addresses. If you send FuelStop a signed transaction that is not included in your address-list, FuelStop will behave no differently than a regular Ethereum node and will broadcast it to the network. If your address has some ETH but not enough to cover the transaction's fees, FuelStop will fund your address so that the transaction can go through.

![Addresses](./diagrams/addresses.svg)

#### Transactions
```